### PR TITLE
dev/core#5440 fix admin-only price field display

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Overview
----------------------------------------
See https://github.com/civicrm/civicrm-core/pull/31063

Before
----------------------------------------
_What is the old user-interface or technical-contract (as appropriate)?_
_For optimal clarity, include a concrete example such as a screenshot, GIF ([LICEcap](http://www.cockos.com/licecap/), [SilentCast](https://github.com/colinkeenan/silentcast)), or code-snippet._

After
----------------------------------------
_What changed? What is new old user-interface or technical-contract?_
_For optimal clarity, include a concrete example such as a screenshot, GIF ([LICEcap](http://www.cockos.com/licecap/), [SilentCast](https://github.com/colinkeenan/silentcast)), or code-snippet._

Technical Details
----------------------------------------
_If the PR involves technical details/changes/considerations which would not be manifest to a casual developer skimming the above sections, please describe the details here._

Comments
----------------------------------------
_Anything else you would like the reviewer to note_

### CODE REVIEW

File: templates/CRM/Price/Form/PriceSet.tpl [hunk 1]
```diff
@@ -27,7 +27,7 @@
 
     {foreach from=$priceSet.fields item=element key=field_id}
         {* Skip 'Admin' visibility price fields WHEN this tpl is used in online registration unless user has administer CiviCRM permission. *}
-        {if $element.visibility EQ 'public' || ($element.visibility EQ 'admin' && $adminFld EQ true) || $context eq 'standalone' || $context eq 'advanced' || $context eq 'search' || $context eq 'participant' || $context eq 'dashboard'}
+        {if $element.visibility EQ 'public' || $element.visibility !== 'admin' || $isShowAdmin}
```
Comment (demeritcowboy) [inline templates/CRM/Price/Form/PriceSet.tpl:30]: $isShowAdmin => $isShowAdminVisibilityFields ?

Comment (eileenmcnaughton) [reply templates/CRM/Price/Form/PriceSet.tpl:30]: argh - I switched to the longer variable name as I went along & obviously didn't loop back around

**Comment by demeritcowboy** [comment]
Have put merge ready in case @awestuk has any comments.

A followup could remove the adminFld vars.

**Comment by eileenmcnaughton** [comment]
@demeritcowboy good point re the `adminFld` - I pushed in a removal

**Pull request merged**